$MirOS: ports/www/dillo/v2/patches/patch-doc_dillo_1,v 1.2 2010/02/21 20:08:41 bsiegert Exp $
--- doc/dillo.1.orig	Sun Jun  7 03:40:24 2009
+++ doc/dillo.1	Mon Aug 24 16:30:22 2009
@@ -65,29 +65,29 @@ URL of proxy to send HTTP traffic throug
 .I dpid 
 Dillo plugin daemon
 .TP
-.I ~/.dillo/bm.txt 
+.I ~/.etc/dillo/bm.txt 
 User bookmarks
 .TP
-.I ~/.dillo/cookies.txt 
+.I ~/.etc/dillo/cookies.txt 
 Stored cookies
 .TP
-.I ~/.dillo/cookiesrc 
+.I ~/.etc/dillo/cookiesrc 
 Cookie settings
 .TP
-.I ~/.dillo/dillorc 
+.I ~/.etc/dillo/dillorc 
 Configuration file.
 .TP
-.I ~/.dillo/dpi_socket_dir 
+.I ~/.etc/dillo/dpi_socket_dir 
 Contains name of directory containing dpi sockets.
 .TP
-.I ~/.dillo/dpidrc 
+.I ~/.etc/dillo/dpidrc 
 Contains name of directory containing dpis, and associates
 dpi files with protocols.
 .TP
-.I ~/.dillo/keysrc 
+.I ~/.etc/dillo/keysrc 
 Keybindings.
 .TP
-.I ~/.dillo/style.css 
+.I ~/.etc/dillo/style.css 
 User style sheet
 .SH SEE ALSO
 .BR wget (1)
